charts/orchestrator-k8s/README.md

```diff
@@ -1,7 +1,8 @@
 # Orchestrator-k8s helm chart 
 
 This chart will install the Orchestrator and all its dependencies on kubernetes. 
-It is well suited for dev/QE/test or resource-constrained environments.
+
+<strong><span style="color:RED">THIS CHART IS NOT SUITED FOR PRODUCTION PURPOSES</span></strong>, you should only use it for development or tests purposes
```

Comment:
someone the color is rendered as red:

![Selection_250](https://github.com/parodos-dev/orchestrator-helm-chart/assets/316242/0c53dd14-94ea-4671-be22-776b5f1789b1)

Reply:
ack, I just bold it then